### ISSUE TITLE
Hide bar on specific space

### ISSUE THREAD

**Issue by andreicorpo** [FelixKratz/SketchyBar#42]
Is it possible to hide(toggle visibility of) the bar only on a specifc space?

Usage: I use this awesome bar alongside Yabai and sometimes I want all the possible real estate available on a space(i.e no gaps no paddings) so I would like to be able to hide on a specific space.

Alternatives:
- Hidding the bar on all spaces - this is the best approach I guess, but I use a multi-monitor setup and since this bar is so pretty and has a lot of useful information, I would like to see it on the other monitors.
- Using native fullscreen - this approach blocks me from using yabai features (side by side apps, resizing) and it also creates a dedicated space

**Comment by FelixKratz** [comment]
I think this is not possible currently, but I think it would be simple to achieve, and helpful as well! I have some local commits that improve bar hiding and many other things, maybe I will chuck a feature like this in quickly this evening and then create a new patch release.
How do you like a syntax like:
```bash
sketchybar -m config hidden <on/off/toggle> [optional: <space-id> or <current> for current space]
```
then you could have a hotkey that toggles the bar on the currently selected space?

**Comment by andreicorpo** [comment]
That syntax would be awesome! Especially the `current` idea. Brilliant!

**Comment by FelixKratz** [comment]
I have implemented the new individual bar hiding 92c4af90045b09c04a3fac44db717d6c944e88f0. It is not per space, but per display:
```bash
sketchybar -m bar hidden <on/off/toggle> [optional: <display-id> or <current> for current display]
```
Tell me if this is working for you.

Btw: I have renamed the *config* domain into the *bar* domain. The *config* keyword still works however.
And I have changed something about the text rendering, do you notice a difference in sharpness with the HEAD version?

**Comment by andreicorpo** [comment]
Just saw you posted a new release with these updates.

Works great!

Did a compination of yabai querying, signals and I got my desired behaviour!

Thank you so much. You can close the issue.
